src/main/fc/rc.c

```diff
@@ -518,19 +512,20 @@ FAST_CODE_NOINLINE void calculateFeedforward(const pidRuntime_t *pid, int axis)
     const float rcCommandDeltaAbs = fabsf(rcCommand[axis] - prevRcCommand[axis]);
     prevRcCommand[axis] = rcCommand[axis];
 
-    float setpoint = rawSetpoint[axis];
+    const float setpoint = rawSetpoint[axis];
     float setpointSpeed = (setpoint - prevSetpoint[axis]);
     prevSetpoint[axis] = setpoint;
-    float setpointAcceleration = 0.0f;
-
 
     if (axis == FD_ROLL) {
-        DEBUG_SET(DEBUG_FEEDFORWARD, 0, lrintf(setpoint * 10.0f)); // un-smoothed final feedforward
+        DEBUG_SET(DEBUG_FEEDFORWARD, 1, lrintf(setpointSpeed * 0.1f));   // raw setpoint delta
     }
 
-    // attenuators
-    float zeroTheAcceleration = 1.0f;
+    float setpointAcceleration = 0.0f;
+    float feedforward = 0.0f;
     float jitterAttenuator = 1.0f;
+    float zeroTheAcceleration = 1.0f;
```

Comment:
(quite poorly named variable)

Reply:
I've re-factored to use a boolean.  Not entirely sure if the changes are more efficient.  Let me know if it seems better now.